### PR TITLE
Added command to display the already set token on CLI

### LINKED ISSUE #123: Add command to display the already set token on CLI
The command should be `evalai gettoken`

The output should be the token which is set at `~/.evalai/token.json file`.

### PR DESCRIPTION
**Pull request by nileshprasad137**
Fixes #123 

Added a command to get current token . 

Usage `evalai gettoken`

### CODE REVIEW

**Comment by nileshprasad137** [comment]
I haven't ever worked with `click` . I was not sure what to put as param in `def gettoken(gettoken)`. On putting `ctx` some tests were failing. 

Can you check what should be right thing there. This works though!

File: evalai/get_token.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    if not os.path.exists(AUTH_TOKEN_PATH):
+        echo(style("\nThe authentication token json file doesn't exist at the required path. "
+                   "Please download the file from the Profile section of the EvalAI webapp and "
+                   "place it at ~/.evalai/token.json or use evalai -t <token> to add it.\n\n", bold=True))
+    else:
+        with open(AUTH_TOKEN_PATH, 'r') as fr:
+            try:
+                data = fr.read()
+                echo(style("Current token: {}".format(data), bold=True))
```
Comment (RishabhJain2018) [inline evalai/get_token.py]: It should display `The current auth token is {}.format(data)`

File: evalai/main.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 from .add_token import token
 from .submissions import submission
 from .teams import teams
+from .get_token import gettoken
```
Comment (RishabhJain2018) [inline evalai/main.py]: Please change it to `get_token`

File: evalai/main.py [hunk 3]
```diff
@@ -34,3 +35,4 @@ def main(ctx):
 main.add_command(token)
 main.add_command(submission)
 main.add_command(teams)
+main.add_command(gettoken)
```
Comment (RishabhJain2018) [inline evalai/main.py]: Please change it to `get_token`

**Comment by nileshprasad137** [comment]
Made the changes

File: evalai/get_token.py (hunk 1 above)
Comment (RishabhJain2018) [reply evalai/get_token.py]: Hey! this is left. :)

File: evalai/get_token.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    if not os.path.exists(AUTH_TOKEN_PATH):
+        echo(style("\nThe authentication token json file doesn't exist at the required path. "
+                   "Please download the file from the Profile section of the EvalAI webapp and "
+                   "place it at ~/.evalai/token.json or use evalai -t <token> to add it.\n\n", bold=True))
+    else:
+        with open(AUTH_TOKEN_PATH, 'r') as fr:
+            try:
+                data = fr.read()
+                echo("Current token is {}".format(data))
```
Comment (RishabhJain2018) [inline evalai/get_token.py]: It should display `The current auth token is {}.format(data)` where data is token only.

Comment (RishabhJain2018) [reply evalai/get_token.py]: @nileshprasad137 Please change the text too!

**Comment by nileshprasad137** [comment]
Sorry I had not seen the edited comment and I'm traveling at the moment so got dealyed.

**Comment by RishabhJain2018** [comment]
No worries :)

**Comment by nileshprasad137** [comment]
Is it fine ?

**Comment by RishabhJain2018** [comment]
Yeah, I'll modify it. Thanks for adding the feature quickly.

**Comment by nileshprasad137** [comment]
If not you may close pr because my laptop is running out of battery.

**Pull request merged**